apps/settings/src/components/PersonalInfo/ProfileSection/ProfilePreviewCard.vue

```diff
@@ -148,7 +148,7 @@ export default {
 			left: 78px;
 			overflow: hidden;
 			text-overflow: ellipsis;
-			word-break: break-all;
+			word-break: break-word;
```

Comment:
Since this is deprecated https://developer.mozilla.org/en-US/docs/Web/CSS/word-break, let's use
```suggestion
			overflow-wrap: anywhere;
```
like in the docs, as `word-break: normal` is the default already